.env-example

```diff
@@ -1,3 +1,4 @@
 DATABASE_URL=
 DEV_DATABASE_URL=postgres://<db_user>:<db_password>@127.0.0.1:5432/dev_db
 TEST_DATABASE_URL=postgres://<db_user>:<db_password>@127.0.0.1:5432/test_db
+NODE_ENV=test
```

Comment:
Hello @sa-ma nice one, I believe this can be set while running the start script, maybe while running `npm run test`, it does not have to be static.